### ISSUE TITLE
Module env - PrgEnv

### ISSUE THREAD

**Issue by david-barker** [UoB-HPC/GW4-Isambard#4]
From DA@Exeter: The module environment isn’t configured the way I was expecting, based on the information from the training day. I was expecting to see several modules called PrgEnv-xxx where xxx is a compiler (cray, gnu, intel). To switch between compiler versions I should just switch the relevant PrgEnv-xxx modules and all other dependencies will be correctly handled. On Isambard I can only see a PrgEnv-cray module (nothing for gnu or Intel compilers). I don’t know whether this is an XC vs. XS environment difference or whether there is something amiss.

**Comment by jrprice** [comment]
Agreed, we've also had early-access users from Bristol expecting there to be a `PrgEnv-Intel`, rather than having to manually load the right modules and invoke the Intel compilers directly.
